src/pages/Main/MainProduct/MainProduct.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const MAIN_PRODUCT = [
+  {
+    id: 1,
+    img: 'product_img_url',
+    brand: 'Nike',
+    product: 'Dunk Low Wolf Grey Rosewood',
+    price: 140000,
+  },
+  {
+    id: 2,
+    img: 'product_img_url',
+    brand: 'Jordan',
+    product: 'Jordan 1 x Travis Scott Retro Low OG SP Black Phantom',
+    price: 990000,
+  },
+  {
+    id: 3,
+    img: 'product_img_url',
+    brand: 'IAB',
+    product: 'IAB Studio Pigment Sweatshirt & Mini Bag Black',
+    price: 139000,
+  },
+  {
+    id: 4,
+    img: 'product_img_url',
+    brand: 'The North Face',
+    product: 'The North Face 1996 Eco Nuptse Jacket Black',
+    price: 275000,
+  },
+];
+
+const MainProduct = () => {
+  return (
+    <>
+      <MainProductList>
+        {MAIN_PRODUCT.map(({ id, img, brand, product, price }) => {
+          const productPrice = price
+            .toString()
+            .replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+
+          return (
+            <MainProductBox key={id}>
+              <MainProductThumb>
+                <img src={img} alt="product" />
```

Comment:
alt text에 product 변수를 그대로 넣어보는건 어때요?
img에 해당하는 product name들이 동적으로 alt text에 바인딩 되는게 더 좋아보입니다!
```suggestion
                <img src={img} alt={product} />
```